api/src/main/java/org/openmrs/module/fgh/mpi/MpiUtils.java

```diff
@@ -177,5 +177,4 @@ private static DataSource getDataSource() {
 		SessionFactory sf = Context.getRegisteredComponents(SessionFactory.class).get(0);
 		return ((SessionFactoryImpl) sf).getConnectionProvider().unwrap(DataSource.class);
 	}
-	
```

Comment:
Revert this too please

Reply:
I still see it

Reply:
Revert this please

Reply:
Done

Reply:
What is done? Have you committed the change? I still see the diff for the file here

Reply:
I committed the file.

Reply:
I think you might not be understanding what am asking you to do, asm asking you to undo all the changes in this file so that no diff appears for it in this PR